tests/security/test_cortx_port_scanner_kubectl_svc.py

```diff
@@ -25,6 +25,8 @@
 
 LOGGER = logging.getLogger(__name__)
 
+
+@pytest.mark.skip(reason="Need to fix with new services framework")
```

Comment:
is this failing as well?

Reply:
This is not failing, skip marker is removed.